src/core.js

```diff
@@ -1483,7 +1483,7 @@ $.extend( $.validator, {
 
 		// https://jqueryvalidation.org/number-method/
 		number: function( value, element ) {
-			return this.optional( element ) || /^(?:-?\d+|-?\d{1,3}(?:,\d{3})+)?(?:\.\d+)?$/.test( value );
+			return this.optional( element ) || /^(((?:-?\d+|-?\d{1,3}(?:,\d{3})+)|-\.\d+)?(?:\.\d+)?)$/.test( value );
```

Comment:
```suggestion
			return this.optional( element ) || /^(?:-?\d+|-?\d{1,3}(?:,\d{3})+)?(?:-?\.\d+)?$.test( value );
```

Shouldn't it literally just be this?